torch/onnx/symbolic_opset9.py

```diff
@@ -1866,6 +1866,20 @@ def hardswish(g, self):
     hardtanh_ = g.op("Div", hardtanh_, g.op('Constant', value_t=torch.tensor(6, dtype=torch.float)))
     return g.op("Mul", self, hardtanh_)
 
+
+@parse_args('v')
+def hardsigmoid(g, self):
```

Comment:
Thanks for your PR! There is `HardSigmoid` op in ONNX, could you check if you can export to that directly? https://github.com/onnx/onnx/blob/master/docs/Operators.md#HardSigmoid

Reply:
Thank you for your review! In the commit 8e1fd82 below I modified the function, and I assume this is what you expect. Could you check if this is it?